src/lib/wallet/useTransferEvents.js

```diff
@@ -14,11 +15,11 @@ const log = logger.child({ from: 'useTransferEvents' })
 const useTransferEvents = () => {
   const userContext = useUserContext()
   const subscriptionRef = useRef(null)
-  const userContextRef = useRef(userContext)
+  const [getContext] = useRealtimeProps(userContext)
```

Comment:
what does userealtimeprops do?

Reply:
it's a helper wrapping ref-trick used before.

- it creates ref for each chaning var passed,
- updates it by useEffect
- returns memo'ized array of getters returning current value from context

the purpose is to make the effect hook run strictly once keeping the correct codestyle (e.g. all deps passed)

Reply:
maybe change the name of useRealtimeProps to something more clear that indicate that purpose?

Reply:
@sirpy done